docs/monoDrive_home/Vehicle-Configuration.md

```diff
@@ -1,89 +1,213 @@
+
+## Change the vehicle's color
+<div class="img_container">
+    <video width=650px height=340px muted controls autoplay loop>
+        <source src="http://cdn.monodrive.io/readthedocs/vehicle_color.mp4" type="video/mp4">
+    </video>
+</div> 
+
+**Current colors available**   
+On the monoDrive Editor look for the Content Browser. From the folders select Vehicles and then select CarPaint, you will find the current colors availables for any car.
+
+```
+Carpaint_BabyBlue
+Carpaint_Beige
+Carpaint_Black
+Carpaint_Copper
+Carpaint_DarkGrey
+Carpaint_Gold
+Carpaint_LightBlue
+Carpaint_NavyBlue
+Carpaint_OliveGreen
+Carpaint_Red
+Carpaint_Silver
+Carpaint_White
+```
+When selecting a color programmatically (i.e. using any client) you can assign the color to use for the ego vehicle using the `vehicle.json` with the `body` tag. 
+```
+"body": {
+    "color": "Carpaint_White.Carpaint_White",
+    "type": "/Game/Vehicles/crossover_monoDrive_01.crossover_monoDrive_01_C"
+  }
+```
+## PID Speed Controllers
+### Follower PID
+Each vehicle has a pre-defined PID parameters to control how closely and accurate follow others vehicles. You can tune this parameters.
+
+### Speed Maintain PID
+Each vehicle has a pre-defined PID parameters to control how closely and accurate maintains the desired speed.
+
+## Vehicle Dynamics
+Using the monoDrive vehicles you have access to the Mechanical Setup to control the drivetrain type (4WD,FDW,RDW,etc), the maximum RPM and damping, the transmission's type, as well of the gear ratio if not automatic transmission, you can also set suspension settings and more.
+
+<div class="img_container">
+    <video width=650px height=340px muted controls autoplay loop>
+        <source src="http://cdn.monodrive.io/readthedocs/vehicle_mechanical.mp4" type="video/mp4">
+    </video>
+</div> 
+
+## Vehicle Axis and Rotations
+### Axis
+The vehicle axis are defined as follows:   
+**X:** Points to the front of the car.   
+**Y:** Points to the right of the car.   
+**Z:** Points to the top of the car.   
+
+<div class="img_container">
+    <video width=650px height=340px muted controls autoplay loop>
+        <source src="http://cdn.monodrive.io/readthedocs/axis.mp4" type="video/mp4">
+    </video>
+</div>   
+
+### Rotation
+**Yaw:** Positive angle when rotating clockwise around the Z axis and negative counterclockwise.   
+**Pitch:** Negative angle when rotating clockwise around the Y axis and positive counterclockwise.   
+**Roll:** Positive angle when rotating clockwise around the X axis and negative counterclockwise.   
+
+<div class="img_container">
+    <video width=650px height=340px muted controls autoplay loop>
+        <source src="http://cdn.monodrive.io/readthedocs/vehicles_ypr.mp4" type="video/mp4">
+    </video>
+</div> 
+
+## Initial Conditions
+When configuring the ego vehicle using the C++ client or the Python client. You can assign the initial conditions for the ego vehicle:   
```

Comment:
The user can assign the initial conditions for the ego vehicle in the C++ Client or the Python Client in the ego vehicle configuration.